packages/api/database/migrations/1595452149355_technology_schema.js

```diff
@@ -38,9 +38,9 @@ class TechnologySchema extends Schema {
 			table.text('risks');
 			table.text('contribution');
 			table
-				.string('status')
-				.notNullable()
-				.defaultTo('pending');
+				.enu('status', ['pending', 'in review', 'rejected', 'published'])
```

Comment:
Antecipando possíveis problemas futuros, creio que seja melhor termos os status como palavras únicas para tratarmos eles como uma espécie de slug:
```suggestion
				.enu('status', ['pending', 'in_review', 'rejected', 'published'])
```